client/src/app/unauthorized.component.tsx

```diff
@@ -1,25 +1,21 @@
 import { FC, ReactElement } from 'react';
 
-import { useNavigate } from 'react-router-dom';
+interface Props {
+  goBack?: () => void;
```

Comment:
This shouldn't be optional, as it is always being passed.

Reply:
This is optional because the first test doesn't need the goBack props. Do I need to refactor the first test with goBack props?

Reply:
Yeah I'd pass it in every test. I used to only pass the props needed for each test, but I realised that having missing props in tests could lead to unexpected behaviour, so now I add all props in all tests.